src/control/SConscript

```diff
@@ -147,9 +147,18 @@ def scons():
     gosrc = Dir('.').srcnode().abspath
     cartpath = Dir('#/src/cart/src/include').srcnode().abspath
 
-    denv.AppendENVPath("CGO_LDFLAGS",
-                       denv.subst("-L$BUILD_DIR/src/cart/src/gurt"), sep=" ")
-    denv.AppendENVPath("CGO_CFLAGS", "-I%s" % cartpath, sep=" ")
+    prereqs.require(denv, 'ofi', 'hwloc')
+
+    cgolibdirs = denv.subst("-L$BUILD_DIR/src/cart/src/gurt "
+                            "-L$PREFIX/lib64 "
+                            "-L$HWLOC_PREFIX/lib -L$OFI_PREFIX/lib "
```

Comment:
Two coments

1. We should consider using $_LDFLAGS instead like we do with $_RPATH...in fact, maybe $_RPATH isn't necessary if we do that
2. HWLOC_PREFIX is no longer defined so this will add /lib to path.   We switched to installed hwloc

Reply:
Done.  Thanks for your additional feedback.